components/server/src/authorization/authorizer.ts

```diff
@@ -178,34 +210,42 @@ export class Authorizer {
     async hasPermissionOnWorkspace(
         userId: string,
         permission: WorkspacePermission,
-        workspaceId: string,
+        workspace: { id: string; organizationId: string | undefined },
```

Comment:
same here -> lets make this explicit and pass in `workspaceId: string` and `organizationId: string`